client/components/web-preview/index.jsx

```diff
@@ -47,8 +56,13 @@ const WebPreview = React.createClass( {
 	getDefaultProps() {
 		return {
 			showExternal: true,
```

Comment:
Ah, ok! :smile: Since I don't want to make this component too aware of the customization experience, what if we only display the button if there is a `previewUrl` set, and when we save we can provide this component with that prop.